lib/paymentrequest.py

```diff
@@ -275,13 +272,8 @@ def send_payment(self, raw_tx, refund_addr):
         payurl = urllib.parse.urlparse(pay_det.payment_url)
         try:
             r = requests.post(payurl.geturl(), data=pm, headers=ACK_HEADERS, verify=ca_path)
-        except requests.exceptions.SSLError:
-            print("Payment Message/PaymentACK verify Failed")
-            try:
-                r = requests.post(payurl.geturl(), data=pm, headers=ACK_HEADERS, verify=False)
-            except Exception as e:
-                print(e)
-                return False, "Payment Message/PaymentACK Failed"
+        except requests.exceptions.RequestException as e:
+            return False, str(e)
```

Comment:
Now catching the correct exception type: `SSLError` is too narrow, `Exception` is too broad.

Reply:
I didn't really test this but this reads correct.